src/NuGet.Services.EndToEnd/EnvironmentSettings.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+
+namespace NuGet.Services.EndToEnd
+{
+    public static class EnvironmentSettings
```

Comment:
if there is any code duplicated from FunctionalTests, please add a comment on the class, so we will know what needs to be consolidated